### ISSUE TITLE
[KAKU][0.8.39][App v1.1.6 ] Debouncer doesn't work / flows trigger 2 or more time consequent 

### ISSUE THREAD

**Issue by Dijker** [athombv/homey#702]
**What is your Homey version (Settings → System)?**
0.8.39 / KAKU 1.1.6
**What did you try to achieve?**
Use some KAKU devices in flows as a trigger

*Please include detailed steps how we can reproduce the issue*
Flow with Trigger: (any of these) 
WallSwitch On  (AWST8802)
or DoorBell (ACDB-7000)
or Motion Detected (APIR-2150)

Then: 
Increment BetterLogic Value +1 
and/or 
Send Push notification to Android Mobile device

**What was the result?**
Every manual Trigger it increments the value with 2 (viewed in dashboard)
and/or I receive 2 msg on my phone. 
Testing the flow (Test button on Web page) just works as expected and increments with 1 and sends msg once.

**What did you expect as result?**
A debouncer that works and filters the KAKU 433 re-transmits within the 2 or 3 seconds after receiving the first signal. 
 
**Any other remarks**
Didn't check it for the other transmitting devices (AMST-606 or AYCT-102) but I expect the same there.

**Comment by Dijker** [comment]
Even this flow is triggered twice by the APIR-2150 Motion Detected:

![image](https://cloud.githubusercontent.com/assets/12412734/16318465/a2008cd4-398f-11e6-9494-8b74f8e924a6.png)

although I would think my CountDown app- debouncer prevents triggeriging a second time within the 30 seconds after a trigger. 
This debouncer tweak worked for me before to fix the doorbell.

**Comment by Dijker** [comment]
Strange, didn't see updates but after two days testing again everything works as expected....
tested several times and can't reproduce the problems. 
so closing for now.

**Comment by edtst** [comment]
This problem has been re-introduced since 0.92.

**Comment by Dijker** [comment]
@edtst
 I tested it today after your msg, 
I don't see that  problem here.... tested it with the simplest  flow to increment  BL Counter on each key-press
Tested for a Bell ACDB-7000A  and the two switches of a AWST8802 . more than 20 times each. never incremented with 2 ....

**Comment by edtst** [comment]
I have an ACDB-7000BC, maybe there is a difference.

**Comment by Dijker** [comment]
@edtst do you have other KAKU transmitters to test ? or another homey anywhere?
Question is: is it the ACDB-7000B (generic) or your device.... or Homey or your Homey Configuration?

**Comment by edtst** [comment]
I have many KaKy devices, but they are on/off. They work fine. No othe rHomey though.
The doorbell rings once on the 3 receivers I have. Only Homey picks it up twice. 
It worked fine before the 92 update.

**Comment by Dijker** [comment]
Dis you test other transmitters with homey if  Homey receives the signal once or more than once?

**Comment by edtst** [comment]
Since they are on/off, I have no way of testing. I only have one doorbell. It does not happen with my Somy system.

**Comment by Dijker** [comment]
of course you can test, I tested my AWST 8802!
. create for each button a check with fe a better logic counter of push message to check!

**Comment by edtst** [comment]
You're right. I tested a few of my KaKu modules and they all fire once (with Homey sending or an external remote control). So it's doorbell related.

I removed both the doorbell and the flow and recreated the whole thing. Then it works and it only fires once.

Repeating that step sometimes recreates the problem. So there is something going on.

Mind you, I use the copy method and not the generate method to create the doorbell.
